presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -2869,6 +2869,22 @@ public void testCreateAndInsert()
         assertEqualsIgnoreOrder(actualAfterTransaction, expected);
     }
 
+    @Test
+    public void testRenameView()
```

Comment:
`TestHiveIntegrationSmokeTest` runs against file metastore.
for this type of change it would be good to run a test against thrift metastore(s) too.